src/styles/blocks/head-block/header.scss

```diff
@@ -0,0 +1,19 @@
+.header {
+  position: relative;
+  display: flex;
+  justify-content: flex-end;
+  width: 100%;
+  padding: 0 50px;
+  background-color: $background_white-color;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  @media (max-width:1024px) {
```

Comment:
The media query value could be abstracted into a variable for better maintainability and consistency across different parts of the codebase.